pkg/collect/logs.go

```diff
@@ -117,7 +117,19 @@ func getPodLogs(ctx context.Context, client *kubernetes.Clientset, pod corev1.Po
 		podLogOpts.TailLines = &limits.MaxLines
 	}
 
-	if limits != nil && limits.MaxAge != "" {
+	if limits != nil && limits.SinceTime != "" {
+		t, err := time.Parse(time.RFC3339, limits.SinceTime)
+		if err != nil {
+			logger.Printf("unable to parse --since-time=%s\n", limits.SinceTime)
+		}
+
+		sinceTime := metav1.NewTime(t)
+		podLogOpts.SinceTime = &sinceTime
+
+	} else if limits != nil && (limits.MaxAge != "" || limits.Since != "") {
+		if limits.Since != "" {
+			limits.MaxAge = limits.Since
```

Comment:
why do we need `Since` if it is the same functionality as `MaxAge`? that feels redundant.